bin/GameIndex.dbf

```diff
@@ -40995,10 +41004,6 @@ Name   = World Series of Poker - Tournament of Champions
 Region = NTSC-U
 Compat = 5
 ---------------------------------------------
-Serial = SLUS-21492
```

Comment:
This was at the correct location, why did you move it ? All you had to do was rename it.